sass/directives/02_base_components/button/_button.scss

```diff
@@ -222,6 +222,101 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+// Fancy Play Button
```

Comment:
this is all so different from everything else in button.scss. I'd leave it in this folder so that it gets documented with buttons in the pattern library, but I'd make a separate file for it. we can work on documentation together